apps/web/src/components/WhyBase/PerformanceAndCost.tsx

```diff
@@ -0,0 +1,97 @@
+import { ButtonWithLinkAndEventLogging } from 'apps/web/src/components/Button/ButtonWithLinkAndEventLogging';
+import { ButtonVariants } from 'apps/web/src/components/Button/Button';
+
+export default async function PerformanceAndCost() {
+  return (
+    <div
+      id="performanceAndCost"
```

Comment:
nitpick: If all these IDs could be in one place, that would avoid potential issue?